### DISCUSSION TITLE
How can organizations get access to Insiders?

### DISCUSSION THREAD

**Discussion by Wolf2323** [squidfunk/mkdocs-material#2109] [Q&A] [answered]
Hello @squidfunk,

previously I sponsored you with my private account (this one) and now we decided to switch the sponsoring to our organization account (https://github.com/BetonQuest).
Now we wonder if it's even possible to give an organization access to a private repository. We obviously want to use the insider version in our CI, Were / are you able to add our organization as a member of the private repo?

**Comment by squidfunk** [comment, marked as answer]
__TL;DR: yes – create a bot account, and send the organization's and bot's name to sponsors@squidfunk.com.__

Thanks for asking! Unfortunately, I cannot add organizations as collaborators – only users – that's just not possible with GitHub. It has already led to some confusion with the need to contact the organization to decide on which user to add 😕 I already added a notice on my [sponsor profile](https://github.com/sponsors/squidfunk#sponsors) that organizations should contact me when sponsoring.

I guess the recommended way is to create a bot account for your org, and make it the owner of the pipelines and tokens. That's at least how I do it with my CI pipelines.

**Comment by Wolf2323** [comment]
What is now the best solution for us. I mean we already paid for a month. So is it maybe possible, that when i now create the Bot account that you add this account, the i stop the sponsoring with the organisation and when one month is over i start sponsoring with the bot account? Or do i need to sponsor twice this month?

**Reply by squidfunk** [reply]
Yes, sure, we can make an exception for that. You don't have to pay twice.

**Reply by Wolf2323** [reply]
The Bot Account is @BetonQuest-Bot

**Reply by squidfunk** [reply]
I added the bot account. You don't have to switch to sponsoring with the bot account, the org is perfectly fine 😊

**Reply by Wolf2323** [reply]
That is nice. Thank you very much.

We love what you do. Keep doing it ;)

**Reply by squidfunk** [reply]
Thanks! I'll select my first reply as the accepted answer, as it yields the relevant advice to future organizations sponsoring.

**Reply by squidfunk** [reply]
That's correct, as you haven't accepted the pending invite. Please check the email account associated with your bot to accept the invite. You will then have access to Insiders, as with your previous account(s).

[a comment by an automated account is not shown]
